Blockzilla/SettingsViewController.swift

```diff
@@ -606,7 +547,7 @@ extension SettingsViewController: SearchSettingsViewControllerDelegate {
     }
 }
 
-private class BlockerToggle {
+class BlockerToggle {
```

Comment:
It's *possible* this should get its own file now that we're using it in multiple View Controllers.

Reply:
Good call.